dev/io.openliberty.jstl.facade/bnd.bnd

```diff
@@ -0,0 +1,52 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jstl.facade
+Bundle-Description: facade to provide a classloader for our internal jstl classes for osgi application jsp backwards compatibility; version=${bVersion}
+
+# Must provide same capability as the com.ibm.ws.jsp.jstl.facade bundle
+Provide-Capability: com.ibm.ws.jsp.jstl.facade
```

Comment:
Need to provide capability for this part: https://github.com/OpenLiberty/open-liberty/blob/43eeef190f08cb845850d8bad1b21ab3c88d5b54/dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/webcontainerext/JSPExtensionFactory.java#L507